src/shapes/Text/Text.ts

```diff
@@ -1780,7 +1820,7 @@ export class Text<
     const parsedAttributes = parseAttributes(element, Text.ATTRIBUTE_NAMES);
 
     const {
-      textAnchor = LEFT,
+      textAnchor = LEFT as typeof LEFT | typeof CENTER | typeof RIGHT,
```

Comment:
Maybe you should add this as top level type `HAlign` `VAlign`

Reply:
meh, parseAttributes needs to be typed at some point.
Not wasting time on it just for this particular use case